lib/alert.py

```diff
@@ -30,10 +30,20 @@ def sent_date(self):
     def expires_date(self):
         return AlertDate(self.expires)
 
+    @property
+    def is_current_public(self):
+        return self.is_current and self.is_public
+
+    @property
+    def is_expired_or_test(self):
+        return self.is_expired or not self.is_public
```

Comment:
That's not quite true: an alert can also (potentially) be not started. We lose this property in a later commit anyway, but in principle we should be as explicit as we can to avoid future subtleties.